<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDialogTestForm</class>
 <widget class="QDialog" name="BaseDialogTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Base Dialog Test</string>
  </property>
  <widget class="QPushButton" name="save_new_button">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>335</y>
     <width>119</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Close the form, any unsaved changes will be lost</string>
   </property>
   <property name="text">
    <string>Save/New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_done_button">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>335</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Save the updated Conditions List</string>
   </property>
   <property name="text">
    <string>Save/New</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="order_table">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>621</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>561</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Copyright Lorn B Kerr, Version 1.0.0</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>save_done_button</tabstop>
  <tabstop>save_new_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
